reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Jobs.md

```diff
@@ -1,371 +1,415 @@
 ---
-description:  Describes how to run background jobs on remote computers. 
+description: Describes how to run background jobs on remote computers.
 keywords: powershell,cmdlet
 Locale: en-US
-ms.date: 12/01/2017
+ms.date: 11/11/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_remote_jobs?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_Remote_Jobs
 ---
-
 # About Remote Jobs
 
-## SHORT DESCRIPTION
-
+## Short Description
 Describes how to run background jobs on remote computers.
 
-## DETAILED DESCRIPTION
+## Detailed Description
+
+PowerShell concurrently runs commands and scripts through jobs. There are three
+jobs types provided by PowerShell to support concurrency.
 
-A background job is a command that runs asynchronously without interacting
-with the current session. The command prompt returns immediately, and you can
-continue to use the session while the job runs.
+- `RemoteJob` - Commands and scripts run in a remote session.
+- `BackgroundJob` - Commands and scripts run in a separate process on the local
+  machine. For more information, see [about_Jobs](about_Jobs.md).
+- `PSTaskJob` or `ThreadJob` - Commands and script runs in a separate thread
+  within the same process on the local machine. For more information, see
+  [about_Thread_Jobs](/powershell/module/ThreadJob/about_Thread_Jobs).
 
-By default, background jobs run on the local computer. However, you can use
-several different procedures to run background jobs on remote computers.
+Running scripts remotely, on a separate machine or in a separate process,
+provide great isolation. Any errors that occur in the remote job do not affect
+other running jobs or the parent session that started the job. But the remoting
+layer adds overhead, including object serialization. All objects are serialized
+and deserialized as they are passed between the parent session and the remote
+(job) session. Serialization of large complex data objects can consume large
+amounts of compute and memory resources.
 
-This topic explains how to run a background job on a remote computer. For
-information about how to run background jobs on a local computer, see
-[about_Jobs](about_Jobs.md). For more information about background jobs, see
-[about_Job_Details](about_Job_Details.md).
+> [!IMPORTANT]
+> The parent session that created the job monitors the job status and collects
```

Comment:
I think "job monitors" looks like a single noun, but it's not intended as such. I'd reword as "The parent session that creates the job also monitors the job status and ..."

Reply:
fixed